--- a/src/qpid/broker/MessageGroupManager.cpp
+++ b/src/qpid/broker/MessageGroupManager.cpp
@@ -280,7 +280,7 @@
          g != messageGroups.end(); ++g) {
         qpid::types::Variant::Map info;
         info[GROUP_ID_KEY] = g->first;
-        info[GROUP_MSG_COUNT] = g->second.members.size();
+        info[GROUP_MSG_COUNT] = (uint64_t) g->second.members.size();
         // set the timestamp to the arrival timestamp of the oldest (HEAD) message, if present
         info[GROUP_TIMESTAMP] = 0;
         if (g->second.members.size() != 0) {
